website/project/model.py

```diff
@@ -2291,6 +2292,14 @@ def update_node_wiki(self, name, content, auth):
         )
         new_page.save()
 
+        wiki_updated.send(
```

Comment:
The first argument to the `send` method is the "sender" of the signal. In this case, the node.  The wiki name can be an additional kwarg; maybe use `wiki_name` for clarity.